docs/contributing/cursorless-in-neovim.md

```diff
@@ -40,7 +40,13 @@ In order to test out your local version of the extension or to run unit tests lo
 
 The debug logs are written in `C:\path\to\cursorless\packages\cursorless-neovim\out\nvim_node.log`.
 
-NOTE: This will spawn a standalone nvim instance that is independent of VSCode. Consequently after you're done debugging, you need to close nvim.
+NOTE: This will spawn a standalone nvim instance that is independent of VSCode. Consequently after you're done
+debugging, you need to close nvim.
+
+### Running lua tests
+
+Their are separate cursorless and lua tests. You can run the lua tests by entering the `cursorless.nvim` folder and
```

Comment:
I know you marked this as resolved but I may want to clarify my thoughts:

- there are Cursorless tests for vscode that test the actions (`take`, `chuck` etc.)
- there are unit tests for Cursorless that tests specific typescipt functions (`toggle decorations`, `visible multiple regions`, etc.). These tests don't rely on vscode or neovim.
- there are Cursorless tests for neovim that test the actions (`take`, chuck`, etc.). 99% of them are the same as the vscode ones except some new ones are needed in order to test all actions, due to no hats (and the fact that lots of vscode tests rely on hats)
- there are unit tests for lua in neovim. These tests are neovim specific and are testing lua functions. They need to be executed in neovim. There is no such tests for vscode because we assume the functions provided by [vscode APIs](https://code.visualstudio.com/api/references/vscode-api) have already been tested and work properly.

When you say `separate cursorless and lua tests` I understand that you mean `neovim cursorless and neovim lua tests`, right? Is that enough explanations? cc @pokey

Reply:
Yep, my feeling was that since the file is about neovim then them being contextual to neovim was implied, but happy to make it more explicit.

Reply:
We could also describe all the different tests into a new documentation page somewhere in Cursorless docs, so we don't overload the neovim instructions above? cc @pokey